smartseq2_single_sample/run_multi_samples/ss2_single_sample_wf.wdl

```diff
@@ -95,7 +95,7 @@ task RsemExpression {
     cut -f 1,4,5 "${rsem_out}.genes.results" >"${rsem_out}.gene.expected_counts"
   }
   runtime {
-    docker: "humancellatlas/rsem"
+    docker: "humancellatlas/rsem:v1.3.0"
```

Comment:
Why does this one have a "v" prefix? Can we just make it "1.3.0" to make it consistent with the others?

Reply:
This image is using RSEM tool, so it is not following semantic versioning, this tag is following the version of RSEM within it. There is another image `humancellatlas/star:2.5.3a-40ead6e` in this same situation. For these specific images, I think it's better to make them consistent with the tools' release version, instead of other general docker images' version.